sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py

```diff
@@ -122,6 +128,11 @@ def _create_handler(self):
         source = Source(self._source)
         if self._offset is not None:
             source.set_filter(self._offset._selector())  # pylint:disable=protected-access
+
+        desired_capabilities = utils.data_factory(
+            types.AMQPArray([types.AMQPSymbol(self._receiver_runtime_metric_symbol)]))\
+            if self._track_last_enqueued_event_info else None
```

Comment:
FWIW I think ternary ops are best when things are short and fit on a single line.  Only a suggestion but something like this is more approachable (to me)
```
desired_capabilities = None
if self._track_last_enqueued_event_info: 
    symbol = self._receiver_runtime_metric_symbol
    desired_capabilities = data_factory(AMQPArray([AMQPSymbol(symbol)]))
```

Reply:
good suggestion, code updated